druid/examples/input_region.rs

```diff
@@ -0,0 +1,219 @@
+// Copyright 2019 The Druid Authors.
```

Comment:
You can change the 2019 to 2022 or 2023, whichever year you created this file on.